src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenRefLocalTests.cs

```diff
@@ -12,6 +12,466 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests
     [CompilerTrait(CompilerFeature.RefLocalsReturns)]
     public class CodeGenRefLocalTests : CompilingTestBase
     {
+        [Fact]
+        public void RefReassignParamByVal()
+        {
+            var comp = CompileAndVerify(@"
+using System;
+class C
+{
+    static void Main()
+    {
+        int x = 0;
+        ref int rx = ref x;
+        int y = 5;
+        Console.WriteLine(rx);
+        M(rx = ref y);
+        Console.WriteLine(rx);
+    }
+
+    static void M(int p)
```

Comment:
>static void M(int p) [](start = 4, length = 20)

Do we have a test for a similar scenario, but when this parameter is ```ref```?

Reply:
Yes, the method is called "AssignInMethodCall"